akvo/rsr/tests/views/test_account.py

```diff
@@ -483,7 +480,7 @@ def test_invalid_password(self):
         response = self.c.post('/auth/register/', json.dumps(data), content_type='application/json', HTTP_X_CSRFTOKEN=csrftoken)
 
         self.assertEqual(400, response.status_code)
-        self.assertTrue(response.content.decode('utf-8').find('Passwords do not match') > 0)
+        self.assertTrue(response.content.decode('utf-8').find("The two password fields didn\\u2019t match") > 0)
```

Comment:
Why the unicode escape? Isn't the apostrophe in ASCII?

Reply:
Fixed by using the serialized JSON response.